SpatialOS/snapshots/default.txt

```diff
@@ -0,0 +1,106 @@
+entity {
```

Comment:
considering that this file is generated, is it necessary to include it?

Reply:
I think it's fine either way. I found it a neat idea to include this file here for "demonstration purposes". Happy to remove it if people feel strongly about it.

Reply:
I don't feel strongly either way. If we decide to keep it as an example, how about documenting this example file in `docs/migration-guide/faq.md`?

Reply:
Good idea. Done.